src/components/base/ConnectionPropertyDrawer.vue

```diff
@@ -0,0 +1,223 @@
+                        </v-row>
+                        <v-row>
+                            <v-card-text>
+                                <v-simple-table dense fixed-header class="table">
+                                    <template>
+                                        <thead>
+                                            <tr>
+                                                <th>Control</th>
+                                                <th>Key</th>
+                                                <th>Value</th>
+                                            </tr>
+                                        </thead>
+                                        <tbody>
+                                            <tr v-for="item in spec" :key="item.key">
+                                                <td width="200px">
+                                                    <v-slider v-model="item.value" :step="item.step" :max="item.max" :min="item.min"></v-slider>
+                                                </td>
+                                                <td>
+                                                    <code>{{ item.name }}</code>
+                                                </td>
+                                                <td width="125px">
+                                                    <v-text-field v-model="item.value" :step="item.step" type="number" :suffix="item.units"> </v-text-field>
+                                                </td>
+                                            </tr>
+                                        </tbody>
+                                    </template>
+                                </v-simple-table>
+                            </v-card-text>
+                        </v-row>
+                    </v-col>
+                    <v-divider vertical inset></v-divider>
+                    <v-col>
+                        <v-card-text>
+                            <v-row no-gutters>
+                                <v-col cols="2">Source:</v-col>
+                                <v-col cols="5"
+                                    ><v-chip v-if="chip1" close color="green" text-color="white" closable @click:close="chip1 = false">{{ component }}</v-chip></v-col
+                                >
+                            </v-row>
+                            <v-row no-gutters>
+                                <v-col cols="2">Sinks:</v-col>
+                                <v-col cols="5"
+                                    ><v-chip v-if="chip2" close color="green" text-color="white" @click:close="chip2 = false">{{ component }}</v-chip></v-col
+                                >
+                                <v-col cols="5"
+                                    ><v-chip v-if="chip3" close color="green" text-color="white" @click:close="chip3 = false">{{ component }}</v-chip></v-col
+                                >
```

Comment:
This needs to be based on the `v-for` that loops through an array and generates the chips.